media-libs/ilmbase/ilmbase-2.2.0-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit cmake-multilib
+
+DESCRIPTION="OpenEXR ILM Base libraries"
+HOMEPAGE="http://openexr.com/"
+# changing sources. Using a revision on the binary in order
+# to keep the old one for previous ebuilds.
+SRC_URI="https://github.com/openexr/openexr/archive/v${PV}.tar.gz -> openexr-${PV}-r1.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/12" # based on SONAME
+KEYWORDS="~amd64 -arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~x64-macos ~x86-solaris"
+IUSE="namespace-versioning static-libs"
+
+DOCS=( AUTHORS ChangeLog NEWS README )
+
+# Is this really needed anymore? The header files are exactly the same.
+MULTILIB_WRAPPED_HEADERS=( /usr/include/OpenEXR/IlmBaseConfig.h )
+
+S="${WORKDIR}/openexr-${PV}/IlmBase"
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DNAMESPACE_VERSIONING=$(usex namespace-versioning)
+		-DBUILD_SHARED_LIBS=$(usex !static-libs)
+	)
+
+	cmake-utils_src_configure
+}
+
+multilib_src_install_all() {
+	einstalldocs
+
+	# package provides .pc files
+	find "${D}" -name '*.la' -delete || die
```

Comment:
ok if they're somehow created... please check why they are, as CMake does not by itself create `.la` files